src/factory-in-proc.js

```diff
@@ -129,15 +133,16 @@ class FactoryInProc {
     }
 
     const node = new Node(options)
-
-    series([
-      (cb) => options.init
-        ? node.init(cb)
-        : cb(),
-      (cb) => options.start
-        ? node.start(options.args, cb)
-        : cb()
-    ], (err) => callback(err, node))
+    node.exec.once('ready', () => {
```

Comment:
Something isn't quite right here. the node is created but then what we listen on the ready event is `exec`??